cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -39,9 +39,8 @@
 
 {% set nav_items, has_children = get_secondary_nav_items(page, request.site.hostname) %}
 
-{# TODO: This should be hidden on tablets as well. #}
 <nav class="o-secondary-navigation
-            {{ 'u-hide-on-mobile' if not has_children else '' }}"
+            {{ 'o-secondary-navigation__no-children' if not has_children else '' }}"
```

Comment:
I'd prob swap this around to avoid the negated logic:

`{{ '' if has_children else 'o-secondary-navigation__no-children' }}`